src/RedaxtorSeo.js

```diff
@@ -275,4 +275,7 @@ export default class RedaxtorSeo extends Component {
  */
 RedaxtorSeo.__renderType = "BEFORE";
 RedaxtorSeo.__editLabel = i18n.__floatingEditLabel;
-RedaxtorSeo.__name = i18n.__checkboxName;
\ No newline at end of file
+RedaxtorSeo.__name = i18n.__checkboxName;
+RedaxtorSeo.applyEditor = function(node, data){
+
```

Comment:
When component is not really designed for appying to node, it's better to throw an error, rather then silently do nothing.
I.e. add 
`throw new Error("SEO editor data can't be applied to DOM node, update that data on server side")`